src/HybridSolver.cpp

```diff
@@ -666,15 +577,16 @@ void HybridSolver::Mult(const mfem::BlockVector& Rhs, mfem::BlockVector& Sol) co
     cg_->Mult(trueHrhs_, trueMu_);
 
     chrono.Stop();
-    timing_ += chrono.RealTime();
+    timing_ = chrono.RealTime();
 
     if (myid_ == 0 && print_level_ > 0)
     {
         std::cout << "  Timing: PCG done in "
                   << timing_ << "s. \n";
     }
 
-    num_iterations_ += cg_->GetNumIterations();
+    // TODO: decide to use = or += here and in timing_ update (MinresBlockSolver uses +=)
```

Comment:
Perhaps total iterations should be tracked separately. Or maybe not at all and let the user take care of it.

Reply:
I think it's more like the user's job to keep track of the total iteration/time if the solver is called in a loop.